script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -781,123 +843,136 @@ It's quite ironic.
 [sha:ff9084d428cea6b7187b8617eaa9583db7c21410]{
 -- Page 694, Offset 16541.
 -- 無限に広がる人間を恐怖した男が、その解決として、無限の時間を求めたのだ。
-But the man who had so feared the infinitely expanding humanity, found his solution in the pursuit of infinite time.
+The man, who so feared the infinite expansion of humanity,
+ found his solution in the pursuit of infinite time.
 }
 [sha:b4c3e764230baee58c336b47e98be4ac8278fa10]{
 -- Page 695, Offset 16542.
 -- 『……ふうん。まあ、ここらで足抜けするのも悪くない判断だよ。例の改革はともかく、<印刷技術|グーテンベルグ>の一般化が致命的だからな。しばらく荒れるだろ、<教|う><会|ち>も協会も。
-// lint-off:LintPageOverflow
-// possibly the longest page in the game, needs desperate help
-"Hmm... well, breaking your ties with us now isn't a bad call. The rapid spread of the Gutenberg press has proven critical, not to mention the whole reformation thing. It's gonna get rough for a while, both for us of the Church, and the Association.
+// Unvoiced
+// Possibly the longest page in the game, edit at your peril
+// Ruby used to cram more shit in, forgive me
```

Comment:
If only monospacing didn't exist

Reply:
me when i discover the guy responsible for monospacing
https://youtu.be/tadr-dG8EzQ